### ISSUE TITLE
Feature Request: pin important messages for quick acess

### ISSUE THREAD

**Issue by alokmahor** [DrKLO/Telegram#884]
In group chat or even chat with user when are many many messages its difficult to find some important message. for that we have to scroll a lot. 

there should be a feature to pin message for that user chat and group chant. so one can see pinned messages per user/group

**Comment by VBelozyorov** [comment]
You can use any #tag for this purpose, isn't you?

**Comment by alokmahor** [comment]
#tag is not proper alternative for pinned messages

**Comment by dkadashev** [comment]
+1 to this feature request

**Comment by iguryanov** [comment]
yep, this one could be useful. Sometimes I keep a chat for a month of talking, for example with wife. 
And from time to time she sends some info I would want to look up fast. Search may not work if I don't remember words or it's an image

**Comment by gnanaswaroop** [comment]
+1
